src/plugins/discover/server/ui_settings.ts

```diff
@@ -211,7 +211,16 @@ export const getUiSettings: () => Record<string, UiSettingsParams> = () => ({
     description: i18n.translate(
       'discover.advancedSettings.discover.showFieldStatisticsDescription',
       {
-        defaultMessage: `Enable "Field statistics" table in Discover.`,
+        defaultMessage: `Enable {fieldStatisticsDocs} in Discover to explore the fields in your data. This functionality is in beta and is subject to change. `,
+        values: {
+          fieldStatisticsDocs:
+            `<a href="https://www.elastic.co/guide/en/kibana/master/show-field-statistics.html"
```

Comment:
Is there a doc links service that can be used for Discover settings? Hardcoded URLs, with the version hardcoded like `master` aren't normally used are they?

Reply:
Has the docs page not been created yet? I just get a 404.

Reply:
It's not merged yet, but we have to create this PR to get the screenshot for the Docs PR.

Reply:
> Is there a doc links service that can be used for Discover settings? Hardcoded URLs, with the version hardcoded like `master` aren't normally used are they?

Talked to Lisa and the `docLinks `service is indeed the recommended way moving forward, but it looks like the service is not available yet for server side/these UI settings. Here's the issue for it https://github.com/elastic/kibana/issues/95389